x/concentrated-liquidity/incentives_test.go

```diff
@@ -1600,6 +1649,24 @@ func (s *KeeperTestSuite) TestInitOrUpdatePositionUptimeAccumulators() {
 			expectedInitAccumValue:   uptimeHelper.twoHundredTokensMultiDenom,
 			expectedUnclaimedRewards: uptimeHelper.emptyExpectedAccumValues,
 		},
+		"(lower < upper < curr) nonzero uptime trackers (position already existing)": {
+			positionLiquidity: DefaultLiquidityAmt,
+			lowerTick: tick{
+				tickIndex:      -50,
+				uptimeTrackers: wrapUptimeTrackers(uptimeHelper.hundredTokensMultiDenom),
+			},
+			upperTick: tick{
+				tickIndex:      50,
+				uptimeTrackers: wrapUptimeTrackers(uptimeHelper.threeHundredTokensMultiDenom),
+			},
+			existingPosition:         true,
+			addToGlobalAccums:        uptimeHelper.threeHundredTokensMultiDenom,
+			positionId:               DefaultPositionId,
+			currentTickIndex:         51,
+			globalUptimeAccumValues:  uptimeHelper.fourHundredTokensMultiDenom,
+			expectedInitAccumValue:   uptimeHelper.twoHundredTokensMultiDenom,
+			expectedUnclaimedRewards: uptimeHelper.emptyExpectedAccumValues,
```

Comment:
How is this calculated? Could you please add comments?

Reply:
Added more comments here for clarity! https://github.com/osmosis-labs/osmosis/pull/5442/commits/bb8298df493789d086491dae215ddc9ceadb84eb